charts/workflow/values.yaml

```diff
@@ -51,6 +51,16 @@ global:
   host_port: 5555
   # Prefix for the imagepull secret created when using private registry
   secret_prefix: "private-registry"
+  # The public resolvable hostname to build your cluster with.
+  #
+  # This will be the hostname that is used to build endpoints such as "deis.$HOSTNAME"
+  hostname: ""
```

Comment:
What happens if `experimental_native_ingress: true`, but the `hostname` field here wasn't updated? Ideally that would lead to a hard-stop error before `helm` actually installs the chart, but I don't know off-hand how to accomplish that.